Dialog/TilesetEditDialog.cpp

```diff
@@ -772,12 +802,17 @@ void TilesetEditDialog::on_pushButton_ImportTile8x8Graphic_clicked()
         QMessageBox::critical(this, QString("Error"), QString("Cannot open file!"));
         return;
     }
+    if(!gfxbinfile.size())
+    {
+        QMessageBox::critical(this, QString("Error"), QString("File size is 0!"));
+        return;
+    }
     tmptile8x8data = gfxbinfile.readAll();
     tmptile8x8data_final = gfxbinfile.readAll(); // Init
     gfxbinfile.close();
 
     // Check size
-    if(tmptile8x8data.size() % 8)
+    if(tmptile8x8data.size() & 7)
     {
         QMessageBox::critical(this, QString("Error"), QString("Illegal file size!"));
```

Comment:
Let's be more descriptive so the user knows what is a legal file size (multiple of 8)

Reply:
Done.